test/unit/sorted-views.spec.js

```diff
@@ -391,4 +391,30 @@ describe('collection/composite view sorting', function() {
       });
     });
   });
+
+  describe('when using `{ reorderOnSort: true }`', function () {
+    beforeEach(function () {
+      this.collectionView = new this.CollectionView({
+        childView: this.ChildView,
+        collection: this.collection,
+        reorderOnSort: true
```

Comment:
`+      expect(this.collectionView.reorder).to.have.been.calledOnce;`

is already there. Did you mean something else ?